src/enhanced-switch.jsx

```diff
@@ -12,6 +11,71 @@ import Paper from './paper';
 import getMuiTheme from './styles/getMuiTheme';
 import warning from 'warning';
 
+function getStyles(props, state) {
+  const {
+    baseTheme,
+  } = state.muiTheme;
+
+  let switchWidth = 60 - baseTheme.spacing.desktopGutterLess;
```

Comment:
these are used in singular places so i think you can replace them with their usage :P